src/js/components/ConfigurationView.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import mixin from 'reactjs-mixin';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import DCOSStore from '../stores/DCOSStore'
+import DescriptionList from './DescriptionList';
+import Service from '../structs/Service';
+
+class ConfigurationView extends mixin(StoreMixin) {
+  constructor() {
+    super();
+
+    this.store_listeners = [{name: 'dcos', events: ['change']}];
+  }
+
+  render() {
+    let {headline, service, versionID} = this.props;
+    let configuration = service.getVersions().get(versionID);
+
+    if (!configuration) {
+      DCOSStore.fetchServiceVersion(service.getId(), versionID);
+
+      return (
+        <div className="container container-pod text-align-center vertical-center inverse">
+          <div className="row">
+            <div className="ball-scale">
+              <div />
+            </div>
+          </div>
+        </div>
+      );
+    }
+
+    let headerValueMapping = {
+      'ID': configuration.id,
+      'Command': configuration.cmd,
+      'Constraints': configuration.constraints,
+      'Dependencies': configuration.dependencies,
+      'Labels': configuration.labels,
+      'Resource Roles': configuration.acceptedResourceRoles,
+      'Container': configuration.container,
+      'CPUs': configuration.cpus,
+      'Environment': configuration.env,
+      'Executor': 'todo',
+      'Health Checks': configuration.healthChecks,
+      'Instances': configuration.instances,
+      'IP Address': configuration.ipAddress,
```

Comment:
Yeah, just as in the current Marathon UI version...

<img width="1421" src="https://cloud.githubusercontent.com/assets/647035/15301747/383d4e16-1b64-11e6-81e9-f7ce8255074f.png">